polaris-react/src/components/LegacyCard/LegacyCard.scss

```diff
@@ -26,12 +22,23 @@
     border-radius: var(--p-border-radius-2);
 
     #{$se23} & {
-      border-radius: var(--p-border-radius-3);
+      @include shadow-bevel(
+        $boxShadow: var(--p-shadow-xs),
+        $borderRadius: var(--p-border-radius-3)
+      );
     }
   }
 
   @media print {
     box-shadow: none;
+
+    #{$se23} & {
+      @include shadow-bevel(
+        $boxShadow: none,
+        $borderRadius: none,
+        $content: none
+      );
+    }
```

Comment:
Shadow bevel successfully removed. However, `Button` styles look off to me. cc @aveline @sarahill @tjonx

<img width="1026" alt="Screenshot 2023-07-06 at 7 40 05 PM" src="https://github.com/Shopify/polaris/assets/32409546/f5031a92-cfae-403c-a3c9-cbdabb00ad95">